diff --git a/python/tvm_ffi/cython/device.pxi b/python/tvm_ffi/cython/device.pxi
index 2eb36fc..25b577e 100644
--- a/python/tvm_ffi/cython/device.pxi
+++ b/python/tvm_ffi/cython/device.pxi
@@ -128,6 +128,7 @@ cdef class Device:
         "hexagon": DLDeviceType.kDLHexagon,
         "webgpu": DLDeviceType.kDLWebGPU,
         "trn": DLDeviceType.kDLTrn,
+        "c_static": DLDeviceType.kDLCPU,
     }
 
     def __init__(self, device_type: str | int, index: Optional[Integral] = None) -> None:
